spec/factories/model.rb

```diff
@@ -2,13 +2,13 @@
 FactoryBot.define do
 ### Airplane
   factory :airplane do
-    make { Faker::Company.name }
-    model { Faker::Company.unique.name }
+    make_model
+    make { make_model.make }
+    model { make_model.model }
     reg do
       Faker::Number.between(from: 100, to: 999).to_s +
         Faker::Name.initials(number: 2)
     end
-    make_model
   end
 ### MakeModel
   factory :make_model do
```

Comment:
The airplane model currently searches and associates, before save, a make_model to relate to the make and model in its own fields. Thus the `make_model` association from the factory was replaced when the airplane was saved.